package.json

```diff
@@ -53,7 +53,8 @@
     "react": "^0.14.0 || ^15.0.0"
   },
   "dependencies": {
-    "attr-accept": "^1.0.3"
+    "attr-accept": "^1.0.3",
+    "prop-types": "^15.5.7"
```

Comment:
Should this go to `devDependencies` and `peerDependencies` since it's React's dependency and not ours?